packages/babel-generator/src/generators/statements.ts

```diff
@@ -124,27 +135,39 @@ export function DoWhileStatement(this: Printer, node: t.DoWhileStatement) {
   this.semicolon();
 }
 
-function buildLabelStatement(prefix, key = "label") {
-  return function (this: Printer, node: any) {
-    this.word(prefix);
+function printStatementAfterKeyword(
+  this: Printer,
```

Comment:
We can make the printer a normal parameter (instead of `this`) so that we don't need to `.call` to use this function.